examples/benchmarks/arewefastyet/bounce.effekt

```diff
@@ -0,0 +1,106 @@
+import io/args
+import text/string
+import mutable/array
+
+record Ball(
+  x: Double,
+  y: Double,
+  xVel: Double,
+  yVel: Double
+)
```

Comment:
TBH, I would model this as

```
interface Ball {
  def bounce(): Bool
}
```
to be closer to the JS version of arewefastyet.

Then create balls of  type `Ball in global` everywhere and use global mutable state.